tests/effect_installed_negative/test_effect_neg.fz.expected_err

```diff
@@ -4,65 +4,68 @@
 ----------------^^^^^
 Callchain that lead to this point:
 
+effect environment 'test_effect_neg.b, test_effect_neg.c' for call to 'test_effect_neg.type.a.type.get' at --CURDIR--/test_effect_neg.fz:37:17:
+  use_a => _ := a.env
```

Comment:
I think this kind of error is very confusing. The user did not call `get` anywhere.